pkg/lib/bundle/validate_test.go

```diff
@@ -73,10 +73,74 @@ func TestValidateBundle_InvalidRegistryVersion(t *testing.T) {
 		logger: logger,
 	}
 
-	err := validator.ValidateBundle(dir)
+	err := validator.ValidateBundleFormat(dir)
 	require.Error(t, err)
 	var validationError ValidationError
 	isValidationErr := errors.As(err, &validationError)
 	require.True(t, isValidationErr)
-	require.Equal(t, len(validationError.AnnotationErrors), 1)
+	require.Equal(t, len(validationError.Errors), 1)
+}
+
+func TestValidateBundleContent(t *testing.T) {
+	logger := logrus.NewEntry(logrus.New())
+
+	validator := imageValidator{
+		logger: logger,
+	}
+
+	var table = []struct {
+		description string
+		mediaType   string
+		directory   string
+		numErrors   int
+		errString   string
+	}{
+		{
```

Comment:
a test case for "no csv"? It might change in the future, but for now that's a requirement.

Reply:
I understand this case but not sure if it is helpful in this specific situation. If there is no CSV, it is considered `PlainType` which we don't validate the contents. So if it is not `RegistryV1` format, the `ValidateBundleContent` func will just return `nil`.